recipes/keep_awake.rb

```diff
@@ -1,80 +1,72 @@
-plist 'disable Power Nap' do
-  path '/Library/Preferences/com.apple.PowerManagement.plist'
-  entry 'DarkWakeBackgroundTasks'
-  value false
+system_preference 'disable computer sleep' do
+  preference :computersleep
+  setting 'Never'
```

Comment:
We should compile a large table / mapping inside `system_preference` that assigns boolean values to preferences instead of requiring users to know usage details such as "On"/"Off" vs "Never", etc. However, this is why things like `keep_awake` exist as a recipe 👍